extensions/notebook/package.json

```diff
@@ -24,6 +24,18 @@
 				}
 			}
 		},
+		{
+			"type": "object",
+			"title": "%notebook.configuration.title%",
+			"properties": {
```

Comment:
Could this be added to the existing properties object in the notebook configuration object above this one? This seems like a duplicate object besides the extra python property.

Reply:
No. I have tested it by including in properties. Got JSON parse error from vs code. Let me change the title to 'notebook.python.configuration.title' so that it would be relevant to python config